### ISSUE TITLE
Add A11Y section

### ISSUE THREAD

**Issue by ollydcuk** [Whitespacers/dev-standards#2]
Incorporating page 11 from current best practice guidelines.

**Comment by neildcuk** [comment]
Pasting this in as a reminder:

**WCAG AA Requirements**

- All images, videos and content-based animations must  have a plain text alternative. Background images and videos, or media that forms part of the visual design should not be textually described and must be hidden from screen reader software.
- Sensory experiences (such as VR/AR, Maps (Streetview), panoramas etc) must have plain text descriptions or explanations as to what content is being shown.
- If CAPTCHA is used, it must have an audio or text alternative. (Recommending: https://www.google.com/recaptcha/intro/index.html#ease-of-use)
- Videos with spoken audio must include either subtitles/captions or transcripts.
- The document must be structured according to the outline model - Heading levels should be adhered to. 
- Colour must not be used as the only visual means of conveying affordance or state on interactive elements. Other stylistic elements must be used in addition to colour. 
- A contrast ratio of at least 4.5:1 must be adhered to on legible text and visual cues. Large-scale text must have a contrast ratio of at least 3:1. Text that forms part of a brand, logo or marque has no minimum contrast requirement.
- Colour must not be used as a sole distinction between individual sets or series of data. Patterns or other visual cues should also be used.
- Interactive interface elements must change in appearance when receiving focus, adhering to the above rules.
- Text containers must support resizing of their text content by up to 2x without compromising the informational structure defined by the layout.
- The entire functionality of any component on a page should be operable by keyboard. A mouse or other pointing device must not be required to consume content. In addition, focus must never be “trapped” on an element via any programmatic means. 
- Time limits (such as those within automatically moving content and/or session timeouts) must be either extendable or removable by user intervention. This includes programmatically scrolling or otherwise shifting content.
- The design must not include automatically moving elements that causes more than three flashes of alternating colour within any second-long period.
- The purpose of links or other calls to action must be determinable from their text content alone (i.e. they cannot contain generic labels such as “click here” or “view”), except where the link is intended to be ambiguous to all users.
- All self-contained navigational elements must remain consistent in both their content and the order of content throughout multiple pages on the site.

There are many other success criteria for AA but these are the main ones that may affect visual design regardless of implementation.
